src/test/config.sol

```diff
@@ -1970,5 +1970,35 @@ contract Config {
             calc_cut:     99_00,
             offboarding:  false
         });
+        afterSpell.collaterals["DIRECT-SPARK-DAI"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      5 * MILLION,
+            aL_gap:       5 * MILLION,
+            aL_ttl:       8 hours,
```

Comment:
[If I got this straight](https://github.com/makerdao/dss-auto-line/blob/bff7e6cc43dbd7d9a054dd359ef18a1b4d06b6f5/src/DssAutoLine.sol#L137), it means that the entire debt ceiling will be available all at once after 8h of the spell being cast. Is this the desired outcome?

I thought the autoline was supposed to be used to gradually increment the debt ceiling.

Why not simply setting the line to 5 million right the way instead of using the autoline?

Reply:
The 5 million can be set right away from autoline as well, no need to wait for the 8 hours.
The idea is to leave the autoline registry already set for when the max line is a greater number.